src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+interface Props {
+  user: User | undefined;
```

Comment:
better to use `null` instead of undefined when your work with objects. `Null` describes that this is an empty object